### DISCUSSION TITLE
Add 50% and other buttons

### DISCUSSION THREAD

**Discussion by sandrich** [DeFiCh/app#1000] [Ideas]
It would be great to have a 50% button and maybe others in the DEX at least. A 50% split is very often used if you want to reinvest into LM.

I guess you could also expand to 10,20,30,40,50,60,70,80,90,100% or 25,50,75,100

**Comment by Solidcryptoblock** [comment]
Wanted to propose the same thing (did on Reddit). It would be nice to implement the same UI features from the mobile Wallet to the Desktop version of the App. 
That includes also:
- Click solution for UTXO-DFI Token Swap
- Dark Theme 

And i hope a security update comes soon because the backupfile is not encrypted and can be exported easily. (Leger,Tresor integrations etc. or at least password-encrypted). 

Happy to see the project moving forward steadily.
